examples/translate/translate.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018-2019 opcua authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/gopcua/opcua"
+	"github.com/gopcua/opcua/debug"
+	"github.com/gopcua/opcua/id"
+	"github.com/gopcua/opcua/ua"
+)
+
+func main() {
+	endpoint := flag.String("endpoint", "opc.tcp://localhost:4840", "OPC UA Endpoint URL")
```

Comment:
Can you order the flags like in the other examples, please?

```go
endpoint := ...
ns := ...
nodePath := ...
flag.BoolVar(...)
```

Also please add the example `device_led.temperature` to the flag description or remove it.